tests/unit/serialization/compact_test.py

```diff
@@ -14,6 +17,13 @@
     FIELD_OPERATIONS,
     _BOOLEANS_PER_BYTE,
     FieldKind,
+    SchemaNotReplicatedError,
+)
+from tests.integration.backward_compatible.serialization.compact_test import (
+    SomeFieldsSerializer,
+    FIELD_KINDS,
+    REFERENCE_OBJECTS,
```

Comment:
I assume some of these imports are now unused

Reply:
Removed at https://github.com/hazelcast/hazelcast-python-client/pull/574/commits/53b1b2f5f916d56afa44207f68d6951697840359.